boofuzz/fuzz_logger_db.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, db_filename, num_log_cases=0):
         self._db_cursor.execute(
             '''CREATE TABLE steps (test_case_index integer, type text, description text, data blob, timestamp TEXT)''')
 
-        self._current_test_case_index = None
+        self._current_test_case_index = 0
```

Comment:
@driechers for the record, did `None` result in an error that you saw?

Reply:
No exceptions but the log for the url does not show up in the dB unless it is set to 0.

Reply:
oops +1 not -1 😆